trestle/core/commands/validate.py

```diff
@@ -51,21 +49,29 @@ def _init_arguments(self):
 
     def _run(self, args):
         """Validate an OSCAL file in different modes."""
-        # get the Model
-        if args[const.ARG_FILE] is None:
+        if args.file is None:
             raise TrestleError(f'Argument "-{const.ARG_FILE_SHORT}" is required')
 
-        model: OscalBaseModel = cmd_utils.get_model(args[const.ARG_FILE])
-        element_paths: List[ElementPath] = cmd_utils.parse_element_args(args[const.ARG_ELEMENT])
+        if args.mode is None:
+            raise TrestleError(f'Argument "-{const.ARG_MODE_SHORT}" is required')
+        mode = args.mode
+        if mode != const.VAL_MODE_DUPLICATES:
+            raise TrestleError(f'Mode value "{mode}" is not recognized.')
 
-        split_plan = self._split_model(model, element_paths)
+        if args.item is None:
+            raise TrestleError(f'Argument "-{const.ARG_ITEM_SHORT}" is required')
+        item = args.item
 
-        try:
-            split_plan.execute()
-        except Exception as ex:
-            split_plan.rollback()
-            raise TrestleError(f'Could not perform operation: {ex}')
+        model = ostarget.TargetDefinition.oscal_read(pathlib.Path(args.file))
```

Comment:
Please load the file content using the utility function. Example code can be found below:
https://github.com/IBM/compliance-trestle/blob/develop/trestle/core/commands/split.py#L66